keylime-agent/src/config.rs

```diff
@@ -696,6 +723,20 @@ fn config_translate_keywords(
         DEFAULT_AGENT_DATA_PATH,
     );
 
+    let mut ima_ml_path = config_get_file_path(
+        "ima_ml_path",
+        &config.agent.ima_ml_path,
+        keylime_dir,
+        DEFAULT_IMA_ML_PATH,
+    );
+
+    let mut measuredboot_ml_path = config_get_file_path(
+        "measuredboot_ml_path",
+        &config.agent.measuredboot_ml_path,
+        keylime_dir,
+        DEFAULT_MEASUREDBOOT_ML_PATH,
+    );
+
```

Comment:
I think using `config_get_file_path()` is not correct here. Note that it will use the provided default path relative from the `keylime_dir` (by default `/var/lib/keylime`).

Maybe a way to fix this would be to change the `DEFAULT_AGENT_DATA_PATH`, `DEFAULT_SERVER_KEY`, `DEFAULT_SERVER_CERT`, etc. to be relative paths (i.e. not starting with `/`) and make `config_get_file_path`  to prepend the `keylime_dir` only when the default value is a relative path.

Reply:
Alternatively, I think it is acceptable that instead of `keylime_dir`, the `/` directory is used in this case.

Also, please add the new options to  `keylime-agent.conf` with a description and default value set. The configuration minor version number should be bumped.

Reply:
"/" is fine. I was thinking of "/sys/kernel/security", just in case anyone would ever want to specify the IMA or MB log file as a relative path. But that's not worth parametrizing, so probably "/" is safer and less involved. OK, let's do that.